styles/styles.css

```
@@ -0,0 +1,316 @@
+    margin-top:51px;
+    margin-bottom:82px;
+    margin-left:371px;   
+    margin-right:371px;
+}
+
+.first-screen-title{
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align:center;
+    text-transform:uppercase;
+    color: #000000;
+}
+
+.first-screen-description{
+    font-weight: 400;
+    font-size:22px;
+    line-height:36px;
+    text-align:center;
+    color: #333333;
+}
+
+
+
+.advantage-title{
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.advantage-description{
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center; 
+    color: #333333;   
+}
+
+.only-here-title{
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.only-here-description{
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #333333;
+}
+
+
+
+.advantage-item:nth-child(2n+1){
+    background-color: rgba(131, 179, 211, 0.12);    
+}
+
+.advantage-item:nth-child(2n+2){
+    background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantage-item:nth-child(-n+2){
+    background-color:#83B3D3;
+}
+
+.advantage-item:nth-child(-n+2) .advantage-title,
+.advantage-item:nth-child(-n+2) .advantage-description {
+    color:#ffffff;
+}
+
+.only-here{
+    background-color: #ffffff;
+}
+
+.only-here-item:nth-of-type(odd){
+    background-color: rgba(131, 179, 211, 0.12); 
+}
+
+.only-here-item:nth-of-type(even){
+    background-color: #ffffff;    
+}
+
+.hotel-search{
+    background-color: #ffffff;
+}
+
+.hotel-search-title{
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;   
+    color: #000000;
+}
+
+.hotel-search-description{
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;    
+    text-align: center;    
+    color: #333333;
+}
+
+.distribution{
+    background-image: url("../images/background-distribution.jpg");
+    background-repeat:no-repeat;
+}
+
+.distribition-title{
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;    
+    text-align: center;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+.distribution-description{
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;    
+    text-align: center;   
+    color: #FFFFFF;
+}
+
+.page-footer{
+    background-color: #ffffff;
+}
+
+.contact-phone{
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 40px;    
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;    
+    color: #000000;
+}
+
+.main-inner{
+    background-color: #ffffff;
+}
+
+.inner-header{
+    background-image: url("../images/catalog-background\ 1.jpg");
+    background-repeat: no-repeat;
+    color: #000000;/*цвет должен быть #ffffff*/
```

Фон можно заставить тянуться.
Про фоновый цвет написал в телеге
```suggestion
  background-size: cover;
  background-color: #324957;
  color: #ffffff;
```